src/test/java/htsjdk/samtools/cram/compression/rans/RansTest.java

```diff
@@ -49,107 +57,145 @@ public String toString() {
         };
     }
 
-    @Test(dataProvider="ransData")
-    public void testRANS(final TestCaseWrapper tc) {
-        roundTripForEachOrder(tc.testArray);
+    public Object[][] getRansTestDataTinySmallLarge() {
+        return new Object[][]{
+                { new TestCaseWrapper(randomBytesFromGeometricDistribution(100, 0.1)), 1, 100 }, // Tiny
```

Comment:
The `TestCaseWrapper` used to be a test **case** wrapper; now its just a test **data** wrapper. Maybe it should be renamed to that, or `TestDataEnvelope`.

Reply:
Also, can you add comments to these data providers above the list of values, saying whats in the returned array, like `// test data, lower limit, upper limit` or something similar.